### PR TITLE
Setup contract development env

### PR DESCRIPTION
**Pull request by tarrencev**
Sets up contracts development environment, configuring ethereum development instance and enabling ES6 + async/await.

### CODE REVIEW

File: packages/contracts/truffle.js [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
- *     provider: function() { 
- *       return new HDWalletProvider(mnemonic, 'https://mainnet.infura.io/<infura-key>') 
- *     },
- *     network_id: '1',
- *     gas: 4500000,
- *     gasPrice: 10000000000,
- *   },
- */
+require("babel-register");
+require("babel-polyfill");
```
[the remark this answers is not in the document]
Comment (simondlr) [reply packages/contracts/truffle.js]: If I have it correctly, sometimes the test helpers require them, right?

[the remark this answers is not in the document]
Comment (simondlr) [reply packages/contracts/truffle.js]: Merging for now regardless to keep the ball rolling! :)

[the remark this answers is not in the document]
Comment (JFrankfurt) [reply packages/contracts/truffle.js]: https://github.com/trufflesuite/truffle/issues/346

[the remark this answers is not in the document]
Comment (simondlr) [reply packages/contracts/truffle.js]: Thanks @JFrankfurt!

**Pull request merged**